apps/cli/src/cli.ts

```diff
@@ -92,6 +93,10 @@ export async function cli(): Promise<void> {
       logger.success("Agent: " + args.message);
       return "User has been informed! If you think you've achieved the goal, execute onGoalAchieved.\nIf you think you've failed, execute onGoalFailed.";
     },
+    "onTimeout": async (args: any) => {
```

Comment:
Since this method is not meant to be used with the client, maybe it'd be better to just pass it as a callback in the Evo constructor